crawl-ref/source/dat/strings/de/items-unique.txt

```diff
@@ -398,7 +398,7 @@ a golden staff
 %%%%
 # sceptre of Asmodeus
 a ruby sceptre
-{n}ein Rubin-Zepter
+{n}ein rubinrotes Zepter
```

Comment:
I don't think ruby is a colour here. It refers to the large ruby on the end of it.
http://crawl.chaosforge.org/Sceptre_of_Asmodeus

Reply:
You're right. In that case "ein rubinbesetztes Zepter" sounds best to me. Or "ein Rubinzepter" for the most literal translation.